src/app/cli/CommandLineInterface.java

```diff
@@ -520,6 +541,295 @@ public void launch(){
                     rate(tokens[1], tokens[2]);
                 }//add more cases
 
+                case PLAY -> {
+                    if(tokens.length <3){
+                        System.out.println("\nUsage:");
+                        System.out.println("PLAY { game | time }");
+                        System.out.println("game            game to be played");
+                        System.out.println("time            time in milliseconds to be added\n"); //Might change later to a more useful format that milliseconds
+                        continue;
+                    }
+                    Game[] game_list = app.search_game_name(tokens[1]);
+
+                    if(game_list.length == 0) {
+                        System.out.println("There is not a game with that name.");
+                        continue;
+                    }
+
+                    Game selected_game = game_list[0];
+                    if(game_list.length > 1) {
+                        System.out.println("Which game would you like to play (enter the number)?");
+                        for(int i = 0; i < game_list.length; ++i) {
+                            Game curr_game = game_list[i];
+                            System.out.println(i+1 + ".\tName: " + curr_game.title() + "\n\tPlay time: " +
```

Comment:
should be put into a `toString`

Reply:
Do you want the game to be printed using it's toString? or make a toString to then use it?